devito/tools/dtypes_lowering.py

```diff
@@ -133,6 +133,19 @@ def dtype_to_cstr(dtype):
 
 def dtype_to_ctype(dtype):
     """Translate numpy.dtype into a ctypes type."""
+    if isinstance(dtype, CustomDtype):
+        return dtype
+
+    # Complex data
+    if np.issubdtype(dtype, np.complexfloating):
+        rtype = dtype(0).real.__class__
+        from devito import configuration
```

Comment:
Can you move this import to the top of the file?

Reply:
no